app/routes/surveys/utils.js

```diff
@@ -82,10 +82,10 @@ export const defaultActiveFrom = (hours: number, minutes: number) =>
     .toISOString();
 
 export const CHART_COLORS = [
-  '#c0392b',
-  '#2b95d6',
-  '#d9822b',
-  '#3dcc91',
+  'var(--lego-link-color)',
+  'var(--color-blue-4)',
+  'var(--color-orange-3)',
+  'var(--color-green-5)',
   '#c73aea',
   '#f4ee42',
   '#98f442',
```

Comment:
Why are only some of these edited?

Reply:
Here, some of the colors were already in the variable sheet. I have now created new variables for the rest of the colors, and applied them here.